pyproject.toml

```diff
@@ -66,9 +67,29 @@ furo = "^2023.9.10"
 mock = "^5.1.0"
 autodoc = "^0.5.0"
 
+[tool.poetry-dynamic-versioning]
+enable = true
+style = "pep440"
+vcs = "git"
+
+[tool.poetry-dynamic-versioning.substitution]
+files = ["pycytominer/__about__.py"]
+
+[tool.commitizen]
```

Comment:
Minor comment: consider keeping the tool configurations together in this file (I noticed Black's appears later after the build-system specification).

Reply:
Good suggestion. Done.